### DISCUSSION TITLE
Issue with feeds

### DISCUSSION THREAD

**Discussion by gandf** [gandf/slick-rss#34] [General]
Some feeds can be misinterpreted.
In this discussion you can send feeds url or OPML data to import your feeds list to show your issues.

Feeds list used to test. Some work and some don't. These are test cases to see if everything works as expected.
If you have any issues with specific feeds, post them in this thread. I would add them to this list.

`<?xml version="1.0" encoding="ISO-8859-1"?><opml version="2.0"><head><title>Slick RSS OPML Export</title></head><body><outline type="rss" text="Tom's Hardware" xmlUrl="https://www.tomshardware.fr/feed/" group="Hardware" excludeUnreadCount="0"/><outline type="rss" text="Next Inpact" xmlUrl="https://www.nextinpact.com/rss/news.xml" group="Informatique" excludeUnreadCount="0"/><outline type="rss" text="Mini Machines" xmlUrl="https://www.minimachines.net/feed" group="Hardware" excludeUnreadCount="0"/><outline type="rss" text="Toile F1" xmlUrl="https://fr.motorsport.com/rss/f1/news/" group="F1" excludeUnreadCount="0"/><outline type="rss" text="sciences et avenir espace" xmlUrl="https://www.sciencesetavenir.fr/espace/rss.xml" group="Sciences" excludeUnreadCount="0"/><outline type="rss" text="sciences et avenir high-tech" xmlUrl="https://www.sciencesetavenir.fr/high-tech/rss.xml" group="Sciences" excludeUnreadCount="0"/><outline type="rss" text="cowcotland" xmlUrl="http://feeds.feedburner.com/cowcotland" group="Hardware" excludeUnreadCount="0"/><outline type="rss" text="Tom's Hardware all" xmlUrl="https://www.tomshardware.com/feeds/all" group="" excludeUnreadCount="0"/><outline type="rss" text="Sexless Friend" xmlUrl="http://www.mangahere.co/rss/sexless_friend.xml" group="" excludeUnreadCount="0"/><outline type="rss" text="deviantART: gallery:meganekkochaser/37582820" xmlUrl="https://backend.deviantart.com/rss.xml?type=deviation&amp;q=by:meganekkochaser+sort:time+meta:all" group="" excludeUnreadCount="0"/><outline type="rss" text="Comics – RandoWis" xmlUrl="http://randowis.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="kukuruyo" xmlUrl="http://kukuruyo.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="Dude and Death : The Origins" xmlUrl="http://tapastic.com/rss/series/24884" group="" excludeUnreadCount="0"/><outline type="rss" text="The Adventures of Business Cat" xmlUrl="http://www.businesscat.happyjar.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="Dragon's Burn" xmlUrl="http://dragonsburn.tumblr.com/rss" group="" excludeUnreadCount="0"/><outline type="rss" text="4-Panel Life" xmlUrl="http://tapastic.com/rss/series/6107" group="" excludeUnreadCount="0"/><outline type="rss" text="The Escapist : Critical Miss" xmlUrl="https://www.escapistmagazine.com/v2/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="the GaMERCaT" xmlUrl="http://tapastic.com/rss/series/526" group="" excludeUnreadCount="0"/><outline type="rss" text="Fisheye Placebo" xmlUrl="http://tapastic.com/rss/series/347" group="" excludeUnreadCount="0"/><outline type="rss" text="deviantART: gallery:yuumei/42604730" xmlUrl="https://backend.deviantart.com/rss.xml?type=deviation&amp;q=by:yuumei+sort:time+meta:all" group="" excludeUnreadCount="0"/><outline type="rss" text="SugarStars" xmlUrl="http://tapastic.com/rss/series/262" group="" excludeUnreadCount="0"/><outline type="rss" text="Ribon" xmlUrl="http://tapastic.com/rss/series/313" group="" excludeUnreadCount="0"/><outline type="rss" text="Megatokyo Comics" xmlUrl="http://megatokyo.com/rss/strips.xml" group="" excludeUnreadCount="0"/><outline type="rss" text="Charlie Sheen" xmlUrl="http://charliesheencomic.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="SPACE BASTARDS" xmlUrl="http://space-bastards.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="Gone With the Blastwave" xmlUrl="http://www.blastwave-comic.com/rss/blastwave.xml" group="" excludeUnreadCount="0"/><outline type="rss" text="Super Effective" xmlUrl="http://www.vgcats.com/super/se.rdf.xml" group="" excludeUnreadCount="0"/><outline type="rss" text="Chaos Life" xmlUrl="http://chaoslife.findchaos.com/feed" group="" excludeUnreadCount="0"/><outline type="rss" text="The GaMERCaT" xmlUrl="http://www.thegamercat.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="FindChaos" xmlUrl="http://www.findchaos.com/feed" group="" excludeUnreadCount="0"/><outline type="rss" text="BladeKitten.com - Most Recent Blade Kitten Comics" xmlUrl="http://feeds2.feedburner.com/bladekittencomics" group="" excludeUnreadCount="0"/><outline type="rss" text="Head Trip" xmlUrl="http://headtrip.keenspot.com/rss.xml" group="" excludeUnreadCount="0"/><outline type="rss" text="Ubunchu!" xmlUrl="http://divajutta.com/doctormo/ubunchu/feed.rss" group="" excludeUnreadCount="0"/><outline type="rss" text="Ctrl+Alt+Del" xmlUrl="http://www.cad-comic.com/rss/rss.xml" group="" excludeUnreadCount="0"/><outline type="rss" text="Aki-chan's Life" xmlUrl="http://www.jimmys-desktop.de/aki.php?rss" group="" excludeUnreadCount="0"/><outline type="rss" text="Manly Guys Doing Manly Things" xmlUrl="http://thepunchlineismachismo.com/feed" group="" excludeUnreadCount="0"/><outline type="rss" text="EVA-303 Updates" xmlUrl="http://eva303.smackjeeves.com/rss/" group="" excludeUnreadCount="0"/><outline type="rss" text="JESUS CHRIST - IN THE NAME OF THE GUN" xmlUrl="http://jesuschriststory.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="The Setup Wizard" xmlUrl="http://thesetupwizard.tumblr.com/rss" group="" excludeUnreadCount="0"/><outline type="rss" text="Digimon World Re:Digitized Translation" xmlUrl="https://digimonworldre.wordpress.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="The World" xmlUrl="http://www.coldbird.net/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="DevZone Topic Subscriptions" xmlUrl="https://devtalk.nvidia.com/rss/feed/tL%2FrW%2BdWpEX7UHUtLNDPsKmaQKaSP6w8udKaQLjmk0A%3D" group="" excludeUnreadCount="0"/><outline type="rss" text="Did You Know Gaming?" xmlUrl="http://didyouknowgaming.com/rss" group="" excludeUnreadCount="0"/><outline type="rss" text="PSO-World.com - The World's Largest Phantasy Star Community - PS Universe, Portable, Zero, Online" xmlUrl="http://www.pso-world.com/rss.xml" group="" excludeUnreadCount="0"/><outline type="rss" text="Break The Bank Habit" xmlUrl="http://brightstarcu.wordpress.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="Vocaloidism" xmlUrl="http://www.vocaloidism.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="PSO2 NA Blog" xmlUrl="http://bumped.org/pso2na/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="Phantasy Star Fan Blog" xmlUrl="http://www.bumped.org/psublog/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="Xenoaisam Website" xmlUrl="http://xenoaisam.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="Mary Cagle Let's Speak English" xmlUrl="http://www.marycagle.com/rss.php" group="" excludeUnreadCount="0"/><outline type="rss" text="Isekai Soul-Cyborg Translations » Feed" xmlUrl="https://isekaicyborg.wordpress.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="The Hazards of Being a Main Character" xmlUrl="https://tapastic.com/rss/series/12603" group="" excludeUnreadCount="0"/><outline type="rss" text="Please Be Merciful" xmlUrl="https://tapastic.com/rss/series/42550" group="" excludeUnreadCount="0"/><outline type="rss" text="The Gamer" xmlUrl="https://www.webtoons.com/en/action/the-gamer/rss?title_no=88" group="" excludeUnreadCount="0"/><outline type="rss" text="The Pigeon Gazette" xmlUrl="https://tapas.io/rss/series/8207" group="" excludeUnreadCount="0"/><outline type="rss" text="Genjicat" xmlUrl="https://tapas.io/rss/series/68320" group="" excludeUnreadCount="0"/><outline type="rss" text="mikiko mini comics" xmlUrl="https://tapas.io/rss/series/4668" group="" excludeUnreadCount="0"/><outline type="rss" text="Soggy Cardboard" xmlUrl="http://www.soggycardboard.com/?feed=rss2" group="" excludeUnreadCount="0"/><outline type="rss" text="Surfacage [and the world will turn to ash]" xmlUrl="http://www.surfacage.net/rss" group="" excludeUnreadCount="0"/><outline type="rss" text="DBZ - The Girl Next Door" xmlUrl="http://the-girl-next-door.smackjeeves.com/rss/" group="" excludeUnreadCount="0"/><outline type="rss" text="Erma" xmlUrl="https://tapas.io/rss/series/23931" group="" excludeUnreadCount="0"/><outline type="rss" text="Watashu" xmlUrl="https://tapas.io/rss/series/33746" group="" excludeUnreadCount="0"/><outline type="rss" text="Coon Comic" xmlUrl="https://tapas.io/rss/series/53898" group="" excludeUnreadCount="0"/><outline type="rss" text="Customer Service Wolf" xmlUrl="http://customerservicewolf.com/rss" group="" excludeUnreadCount="0"/><outline type="rss" text="Crawling Dreams" xmlUrl="https://tapas.io/rss/series/91644" group="" excludeUnreadCount="0"/><outline type="rss" text="The Crawling City" xmlUrl="https://tapas.io/rss/series/42485" group="" excludeUnreadCount="0"/><outline type="rss" text="The Elf is a Freeloader" xmlUrl="https://www.euricette.com/category/series/the-elf-is-a-freeloader/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="dijoninpieces" xmlUrl="https://dijoninpieces.wordpress.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="Omega Harem Translations » Feed" xmlUrl="https://omegaharem.wordpress.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="Fmgandalf translations » Feed" xmlUrl="https://fmgandalf.wordpress.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="dijoninpieces » I Was a Sword When I Reincarnated Category Feed" xmlUrl="https://dijoninpieces.wordpress.com/category/i-was-a-sword-when-i-reincarnated/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="“A Tale of Two Rulers”" xmlUrl="http://figmentforms.tumblr.com/rss" group="" excludeUnreadCount="0"/><outline type="rss" text="Haruka" xmlUrl="https://www.webtoons.com/en/challenge/haruka/rss?title_no=141998" group="" excludeUnreadCount="0"/><outline type="rss" text="Reincarnated As A Dragons Egg" xmlUrl="https://dsrealmtranslations.com/category/reincarnated-as-a-dragons-egg/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="Negative Frames" xmlUrl="https://tapas.io/rss/series/88299" group="" excludeUnreadCount="0"/><outline type="rss" text="Stalker x Stalker" xmlUrl="https://www.webtoons.com/en/challenge/stalker-x-stalker/rss?title_no=245662" group="" excludeUnreadCount="0"/><outline type="rss" text="Future Sound" xmlUrl="https://tapas.io/rss/series/124387" group="" excludeUnreadCount="0"/><outline type="rss" text="Edd Lai's Stories" xmlUrl="https://www.webtoons.com/en/challenge/edd-lais-stories/rss?title_no=301213" group="" excludeUnreadCount="0"/><outline type="rss" text="Deathsitter" xmlUrl="https://www.webtoons.com/en/challenge/deathsitter/rss?title_no=293285" group="" excludeUnreadCount="0"/><outline type="rss" text="London Vampire" xmlUrl="https://www.webtoons.com/en/challenge/london-vampire/rss?title_no=70817" group="" excludeUnreadCount="0"/><outline type="rss" text="Magnolia Online" xmlUrl="https://www.webtoons.com/en/challenge/magnolia-online/rss?title_no=56485" group="" excludeUnreadCount="0"/><outline type="rss" text="Clinic of Horrors" xmlUrl="https://www.webtoons.com/en/challenge/clinic-of-horrors/rss?title_no=274661" group="" excludeUnreadCount="0"/><outline type="rss" text="My Succubus Girlfriend" xmlUrl="https://www.webtoons.com/en/challenge/my-succubus-girlfriend/rss?title_no=116486" group="" excludeUnreadCount="0"/><outline type="rss" text="Vampire Gamer" xmlUrl="https://www.webtoons.com/en/challenge/vampire-gamer/rss?title_no=388506" group="" excludeUnreadCount="0"/><outline type="rss" text="system32comics" xmlUrl="https://www.webtoons.com/en/challenge/system32comics/rss?title_no=235074" group="" excludeUnreadCount="0"/><outline type="rss" text="The Bikini Bottom Horror" xmlUrl="https://stillinthesimulation.com/untitled-comic/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="My Girlfriend's a Hex Maniac" xmlUrl="https://tapas.io/rss/series/124936" group="" excludeUnreadCount="0"/><outline type="rss" text="Little Things PH" xmlUrl="https://tapas.io/rss/series/19771" group="" excludeUnreadCount="0"/><outline type="rss" text="Dungeons &amp; Doodles: Tales from the Tables" xmlUrl="https://tapas.io/rss/series/212513" group="" excludeUnreadCount="0"/><outline type="rss" text="The Weekly Roll" xmlUrl="https://www.webtoons.com/en/challenge/the-weekly-roll/rss?title_no=358889" group="" excludeUnreadCount="0"/><outline type="rss" text="Lore Olympus" xmlUrl="https://www.webtoons.com/en/romance/lore-olympus/rss?title_no=1320" group="" excludeUnreadCount="0"/><outline type="rss" text="Clinic of Horrors (Webtoon Originals)" xmlUrl="https://www.webtoons.com/en/supernatural/clinic-of-horrors/rss?title_no=3414" group="" excludeUnreadCount="0"/><outline type="rss" text="The Magineer" xmlUrl="https://www.themagineer.com/rss/" group="" excludeUnreadCount="0"/><outline type="rss" text="reddit" xmlUrl="http://www.reddit.com/r/hardware/.rss" group="" excludeUnreadCount="0"/><outline type="rss" text="fanlesstech" xmlUrl="https://www.fanlesstech.com/feeds/posts/default" group="" excludeUnreadCount="0"/><outline type="rss" text="youtube" xmlUrl="https://www.youtube.com/feeds/videos.xml?channel_id=UChIs72whgZI9w6d6FhwGGHA" group="" excludeUnreadCount="0"/><outline type="rss" text="thurrott" xmlUrl="https://www.thurrott.com/feed" group="" excludeUnreadCount="0"/><outline type="rss" text="forbes" xmlUrl="http://www.forbes.com/sites/tomcoughlin/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="thememoryguy" xmlUrl="http://thememoryguy.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="feedburner" xmlUrl="http://feeds.feedburner.com/thessdreview/KGyZ?format=xml" group="" excludeUnreadCount="0"/><outline type="rss" text="trendforce" xmlUrl="http://www.trendforce.com/nand-flash/feed.html" group="" excludeUnreadCount="0"/><outline type="rss" text="idc" xmlUrl="https://www.idc.com/rss/idcpressreleases.xml" group="" excludeUnreadCount="0"/><outline type="rss" text="intc" xmlUrl="https://www.intc.com/news-events/press-releases/rss" group="" excludeUnreadCount="0"/><outline type="rss" text="informa" xmlUrl="https://omdia.tech.informa.com/pr/pr-listing/rss/" group="" excludeUnreadCount="0"/><outline type="rss" text="technode" xmlUrl="https://technode.com/rss" group="" excludeUnreadCount="0"/><outline type="rss" text="pcmag" xmlUrl="https://in.pcmag.com/feed.xml" group="" excludeUnreadCount="0"/><outline type="rss" text="anandtech" xmlUrl="http://www.anandtech.com/rss/articlefeed.cfm" group="" excludeUnreadCount="0"/><outline type="rss" text="clubic" xmlUrl="https://www.clubic.com/feed/news.rss" group="" excludeUnreadCount="0"/><outline type="rss" text="zdnet" xmlUrl="https://www.zdnet.fr/feeds/rss/actualites/" group="" excludeUnreadCount="0"/><outline type="rss" text="canardpc" xmlUrl="https://www.canardpc.com/feed" group="" excludeUnreadCount="0"/><outline type="rss" text="techpowerup news" xmlUrl="https://www.techpowerup.com/rss/news" group="" excludeUnreadCount="0"/><outline type="rss" text="techpowerup reviews" xmlUrl="https://www.techpowerup.com/rss/reviews" group="" excludeUnreadCount="0"/><outline type="rss" text="PC Perspective" xmlUrl="https://pcper.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="Computer Hardware Blog" xmlUrl="https://www.computerhardwareinc.com/about/blog/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="Hot Hardware" xmlUrl="https://hothardware.com/mobile/news.aspx" group="" excludeUnreadCount="0"/><outline type="rss" text="Velocity Micro" xmlUrl="https://www.velocitymicro.com/blog/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="tomshardware" xmlUrl="https://www.tomshardware.com/feeds/rss2/all.xml" group="" excludeUnreadCount="0"/><outline type="rss" text="computerhardwareinc" xmlUrl="https://www.computerhardwareinc.com/about/blog/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="poftut" xmlUrl="https://www.poftut.com/category/hardware/computer/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="topflightpc" xmlUrl="https://www.topflightpc.com/blog?format=RSS" group="" excludeUnreadCount="0"/><outline type="rss" text="velocitymicro" xmlUrl="https://www.velocitymicro.com/blog/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="comptoir-hardware" xmlUrl="https://www.comptoir-hardware.com/home.xml" group="" excludeUnreadCount="0"/><outline type="rss" text="modding" xmlUrl="https://www.modding.fr/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="overclocking" xmlUrl="https://overclocking.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="hardwarecooking" xmlUrl="https://www.hardwarecooking.fr/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="techrepublic" xmlUrl="https://www.techrepublic.com/rssfeeds/articles/" group="" excludeUnreadCount="0"/><outline type="rss" text="techrepublic videos" xmlUrl="https://www.techrepublic.com/rssfeeds/videos/" group="" excludeUnreadCount="0"/><outline type="rss" text="techrepublic developer" xmlUrl="https://www.techrepublic.com/rssfeeds/topic/developer/" group="" excludeUnreadCount="0"/><outline type="rss" text="techrepublic hardware" xmlUrl="https://www.techrepublic.com/rssfeeds/topic/hardware/" group="" excludeUnreadCount="0"/><outline type="rss" text="computerweekly" xmlUrl="https://www.computerweekly.com/rss/RSS-Feed.xml" group="" excludeUnreadCount="0"/><outline type="rss" text="intel AI" xmlUrl="https://ai.intel.com/blog/feed" group="" excludeUnreadCount="0"/><outline type="rss" text="intel IoT" xmlUrl="https://blogs.intel.com/iot/feed" group="" excludeUnreadCount="0"/><outline type="rss" text="gamersnexus AMD" xmlUrl="https://www.gamersnexus.net/tag/amd/feed?type=rss" group="" excludeUnreadCount="0"/><outline type="rss" text="microsoft blogs" xmlUrl="http://blogs.microsoft.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="microsoft" xmlUrl="https://news.microsoft.com/feed/" group="" excludeUnreadCount="0"/><outline type="rss" text="apple" xmlUrl="https://www.apple.com/newsroom/rss-feed.rss" group="" excludeUnreadCount="0"/><outline type="rss" text="intelnewsroom" xmlUrl="https://feeds.feedburner.com/intelnewsroom" group="" excludeUnreadCount="0"/><outline type="rss" text="motorsport" xmlUrl="https://www.motorsport.com/rss/f1/news/" group="F1" excludeUnreadCount="0"/></body></opml>`

**Comment by Computurd** [comment]
Hi, the feed is now not working, and I don't know why. Some feeds do not show up at all, and they were working in the past. other feeds show up but will not be marked as read (See image). I am on Chrome version 101.0.4951.54 (Official Build) (64-bit). 
![image](https://user-images.githubusercontent.com/104912846/166699641-9f4970e5-b19b-4962-b1f2-7ac57896ae92.png)

**Reply by gandf** [reply]
Hi,

I'm sorry for issues of the past few days.

Last version is a big change since previous version. _(Manifest V3 from Google remove several features and I had to find bypass methods. For example, there is no longer access to the DOM which interpret HTML in background.)_

I did an update that changes way to identify items. I get item's guid inside feeds if it exists. Otherwise, "title" and "publication date" identifies item.
But reading of guid is not right and all item in feed have same identifier.
I did not see it during my tests because I used feeds list provided in one PR and feeds tested did not have a guid.
I tested with these flows because they had errors with them.

Other issues is that errors are raised with the normal version which do not appear in test mode.
Once published, it looks like some rights are not assigned (to code) or timings are different.

As I upgraded to the V3 manifest, I don't think I can roll back and Google hasn't implemented a rollback.
When I publish new version, it takes 2 to 3 working days before it is published.
Last night I released a 3.5.1 version which will fix this guid issue and should fix bigger issues.
I hope it will be published no later than Friday.

I tested anadtech feed and it should work with next version.
There will be more updates in next few days when I can see what remaining errors are with released version.

On the other hand, if you have feed that never finishes loading, you must delete this feed and add it back for it to work properly.

**In resume, there is no issue on your side, it is latest update that is in question and fix is being deployed.**

**Reply by Computurd** [reply]
Thank you so much for your hard work! This is the most wonderful RSS reader I have ever used, and I look forward to the fix being issued :) Again, thanks for your dedication to this product, I use it every day, and it is great!

**Reply by ghost** [reply]
> On the other hand, if you have feed that never finishes loading, you must delete this feed and add it back for it to work properly.

I have done so many times and nothing has come of it, also the console is still saying there's a GUID error:
`Uncaught (in promise) TypeError: Cannot read properties of null (reading '0')
    at background.js:544:50`

**Reply by gandf** [reply]
New version 3.5.1 is published. Are you still having problems?
Some errors remain present but biggest issues seem to be fixed.

![image](https://user-images.githubusercontent.com/4194210/167014446-2547dd14-b43b-4cce-b593-18e6dea2a2cb.png)

Read with click on title, read with "cross" or send into "read later" work for me.

Do you still have these problems with this new version?

**Reply by gandf** [reply]
Issue with loading some feeds (continuously loading).

**Reply by Computurd** [reply]
I tried to re-import my feeds, but it says this: 
![image](https://user-images.githubusercontent.com/104912846/167031212-795e1f6f-1905-4c7f-a468-581dbd9a2123.png)

**Reply by gandf** [reply]
Version 3.5.2 has just been released. It will allow feeds to be read completely. I tested feeds with guid and without guid.
Test : 93 feeds:
- with Guid
- without Guid
- empty
- with differents errors like 404, ...
- with redirection or not

I think that version 3.5.2 will be deployed no later than Tuesday.

**Reply by gandf** [reply]
If you have exported feeds from previous version, replace all 'excludeUnreadCount=0' by 'excludeUnreadCount="0"'.
Or post text here if you don't mind making your list public.
You can look #41.

**Reply by Computurd** [reply]
I have issues with the following feeds:
http://www.reddit.com/r/hardware/.rss
https://www.fanlesstech.com/feeds/posts/default

**Reply by gandf** [reply]
These 2 feeds will still not work with the latest version.
I will look at what is blocking and fix it.

**Reply by Computurd** [reply]
Thanks, here are a few more that are not working that used to work just fine: 
https://www.youtube.com/feeds/videos.xml?channel_id=UChIs72whgZI9w6d6FhwGGHA
https://www.thurrott.com/feed
http://www.forbes.com/sites/tomcoughlin/feed/
http://thememoryguy.com/feed/
http://feeds.feedburner.com/thessdreview/KGyZ?format=xml
http://www.trendforce.com/nand-flash/feed.html
https://www.idc.com/rss/idcpressreleases.xml
https://www.intc.com/news-events/press-releases/rss
https://omdia.tech.informa.com/pr/pr-listing/rss/
https://technode.com/rss
https://in.pcmag.com/feed.xml

**Reply by gandf** [reply]
Even with the latest changes, there are issues reading these feeds. I look to integrate them in right way.

**Reply by gandf** [reply]
Some feeds are not feed but HTML. I can't get this format for feed's.
For youtube, iframe is used. You must enable iframe in option to show video player.
I replace youtube by youtube-nocookie for privacy and change link to get embedded link.

All these feeds will work except those that don't have supported format.
Some feed can be read (error 404, ...)

Feed logo has been added to top of the page.

Version 3.5.2 has been published out by Google.
I wait tomorrow evening to publish next version to see if any other unseen issues are still present.

**Reply by Computurd** [reply]
![image](https://user-images.githubusercontent.com/104912846/167407026-2d1698e6-9052-4dc5-bb72-486ede34172b.png)

I am on 3.5.2, but some feeds still do not load correctly and it says that access is requested. However, when I click that, it doesn't do anything. How do I fix that?

**Reply by gandf** [reply]
This version is the last published.
Lastest fix for read feeds reading will be published this evening.
Version that read all your feeds is 3.6.

Are iframes enabled?
They must be activated to have the youtube videos but this generates access requests that the extension does not have. These accesses are requested by iframes.

I have to publish new version of this extension to see rights that should be added.
I try not to add too many because the more there are, the longer the reviews by Google before deployment and for security and privacy.

Official versions published on the store have fewer rights and generate more warnings than test version. I don't know how to do it other than deploying.

**Reply by gandf** [reply]
I just uploaded version 3.6.
This version should be good for your feeds.

**Reply by gandf** [reply]
Version 3.6 is published. All feeds work or give error ?

**Reply by Computurd** [reply]
Everything seems to be working perfectly!! Thank you so much for the hard work!!

**Comment by daljr819** [comment]
I;m having issues with links loading in the background option. Even when selected to be on, clicking any link automatically takes me out of Slick and into the new tab

**Comment by gandf** [comment]
Hello,
What browser are you using?
I will try to reread this part of code to see if there can be an explanation.
In my case it works but there is still a random bug in background. And when it happens during use of this function, code stops in place.

**Comment by daljr819** [comment]
I'm using Chrome

**Reply by gandf** [reply]
V3.10 is now online.
Can you export options and feedlist?

Thanks.

**Comment by gandf** [comment]
Can you take a screenshot of your options?
Thanks

**Comment by gandf** [comment]
Verison 3.10 will be deployed in a few days.
Can you export options and feedlist?
This will allow me to test under the same conditions.

Thanks.

**Comment by daljr819** [comment]
[image: image.png]

[image: image.png]

**Comment by gandf** [comment]
I don't see image.png.

In extension management you can se if your version is 3.10.
![image](https://user-images.githubusercontent.com/4194210/202265651-e13a648d-4c89-4c9a-af8c-dfde8dd65d42.png)

In version 3.10, you have two options to export feeds and options in text.
![image](https://user-images.githubusercontent.com/4194210/202265854-fd874549-089c-468f-9af1-6dfb539999f0.png)

**Comment by daljr819** [comment]
My version is  3.010

**Comment by gandf** [comment]
Version 3.10 = Version 3.010

**Comment by daljr819** [comment]
Here is my feed list. Any luck on finding the problem?

**Comment by gandf** [comment]
I think I have reproduced the problem.
As I use the middle button as asual (which opens link in a new tab in background) I had not seen that with right click, it opens link directly.

**Comment by gandf** [comment]
No, right click works on vivaldi but not on chrome.

**Comment by gandf** [comment]
Next version 3.11 will have a fix to open tab in background.

**Comment by gandf** [comment]
Verison 3.11 will be deployed in a few days.

**Comment by gandf** [comment]
Is it working now?

**Comment by daljr819** [comment]
It is. Thanks!
